draft-ietf-rats-architecture.md

```diff
@@ -376,6 +376,73 @@ create Evidence. Examples of environments that can be used as
 Attesting Environments include Trusted Execution Environments (TEE),
 embedded Secure Elements (eSE), or Hardware Security Modules (HSM).
 
+## Layered Attestation Procedures
+
+By definition, the Attester role takes on the duty to create Evidence.
+The fact that an Attester role is composed of several types of environments that
+can be nested or staged adds complexity to the architectural layout of how an
+Attester -- in itself -- is composed and therefore has to conduct the Claims collection
+in order to create believable Attestation Evidence. This complexity is
+independent of the concept defined by Composite Devices.
+Composite Attesters are inherently composed of multiple nested environments.
+The following example is intended to illustrate this composition:
+
+A very common example is elaborated on to illustrate Layered Attestation.
+
+The very first Attesting Environment has to ensure the integrity of
+the (U)EFI / BIOS / Firmware that initially boots up a composite device (e.g.,
+a cell phone). A hardware component, which effectively can encompasses
+immutable execution environments (static code Roots of Trust) and mutable execution environments (dynamic code Roots of Trusts),
+executes the very first Claim Collection. These Claims have to be stored
```

Comment:
The term "stored" sounds like persisted storage, which I don't think is required